src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseConfirmsToProceedRespondentNotificationHandler.java

```diff
@@ -82,7 +82,16 @@ private CallbackResponse notifyRespondentSolicitorForClaimantConfirmsToProceed(C
         }
 
         if (SPEC_CLAIM.equals(caseData.getCaseAccessCategory())
-            && caseData.isLRvLipOneVOne()) {
+            && caseData.isLRvLipOneVOne()
```

Comment:
Could we extract the conditions from the ifs in a separate function? I think there are a lot of conditions in the if and another level of identation below with another if.